artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/messages/ConnectionAbstract.java

```diff
@@ -61,6 +61,7 @@ private ConnectionFactory createAMQPConnectionFactory() {
       }
       JmsConnectionFactory cf = new JmsConnectionFactory(user, password, brokerURL);
       if (clientID != null) {
+         System.out.println("Consumer:: clientID = " + clientID);
```

Comment:
This appears to have been left by mistake. Can you confirm?

Reply:
I am merging without this line

Reply:
Hmmm... actually the Core version had the same print on it.

Reply:
yes, I added the sysout to have this sync with the Core version, this gets logged on the console along with the filer
```
[shsingh@shsingh bin]$ ./artemis consumer --destination topic://Multicast --message-count 10 --clientID test_Demo --durable --protocol amqp
Consumer:: filter = null
Consumer:: clientID = test_Demo
```